proto/zitadel/management.proto

```diff
@@ -279,6 +279,22 @@ service ManagementService {
         };
     }
 
+    rpc GetAllowedLanguages(GetAllowedLanguagesRequest) returns (GetAllowedLanguagesResponse) {
```

Comment:
Why is there a GetAllowedLanguages on the management API? This are instance level settings right?

Reply:
You are absolutely right, I also don't like it like this.

I changed it like this:
- auth and mgmt servers
  - removed GetAllowedLanguages methods
  - deprecated GetSupportedLanguages
- admin server
  - changed permission for GetSupportedLanguages from "iam.read" to "authenticated"
- console
  - use only admin servers GetSupportedLanguages and GetAllowedLanguages

Reply:
works for me if that is usable like this with console